build.sbt

```diff
@@ -103,7 +103,7 @@ publishTo := {
   if (sys.props.isDefinedAt("build.number"))
     Some(Resolver.file("",  new File(Path.userHome.absolutePath+"/.m2/repository")))
   else
-    Some("ripe-snapshots" at "http://nexus.ripe.net/nexus/content/repositories/snapshots")
+    Some("ripe-snapshots" at "https://nexus.ripe.net/nexus/content/repositories/snapshots")
```

Comment:
While we're at this line 40 also generates warning and needs https: `resolvers += "Codehaus Maven2 Repository" at "http://repository.codehaus.org/"`

Reply:
Yeap, fixed.

Reply:
Nice when it's quieter :d